README.md

```diff
@@ -13,13 +13,15 @@ The Front End for the BTI Application journey on GOV.UK
 
 ##### Starting the application:
 
-1) Run [Binding Tariff Classification](https://github.com/hmrc/binding-tariff-classification) on Port 9090: `sbt 'run 9090'`
-2) Start Assets Frontend Using `sm --start ASSETS_FRONTEND -r 3.2.2`
+1) Start Assets Frontend Using `sm --start ASSETS_FRONTEND -r 3.2.2`
```

Comment:
I believe we want to use the latest version of ASSETS_FRONTEND (something like 4.5.0)
Am I right?

Reply:
The front end requires that specific version according to @russellthorn

Reply:
ok

Reply:
Scaffold comes pre-packaged using version 3.2.2, I don't think we're tied to that. Bumping to version 4 may have a knock on effect on styling and may require changes to class names to adopt the same styling due to differences in v3 VS v4. I haven't used v4 much. I did try implement v4 in the early build of imports, but ended up reverting the the latest version of v3 because had some issues with v4 of AF.

In all honesty, There's little v4 gives us over v3, possibly nothing. I don't know off the top of my head. I would suggest if anything, bumping to the latest version of v3 for now. Which has zero risk of requiring any further work and only bumping versions away from v3 when v5 (will be known as HMRC frontend) is released. As I understand it the new version is currently being produced and will be inline and keeping up with the latest assets frontend from GOV.UK, simply called Frontend. Which the new design system is built with. Mainly new class names etc using BEM naming conventions.